monai/transforms/io/array.py

```diff
@@ -514,3 +516,49 @@ def __call__(
             "    https://docs.monai.io/en/latest/installation.html#installing-the-recommended-dependencies.\n"
             f"   The current registered writers for {self.output_ext}: {self.writers}.\n{msg}"
         )
+
+
+class MappingJson(Transform):
```

Comment:
I think `WriteFileMapping` or some other name describes what the class is doing a little better, specifically it states that it writes something.